### PR TITLE
Nicer error message for CSV with no properties.

### PR DESCRIPTION
**Pull request by gianm**
The old message didn't mention that `columns` is an option, which was confusing.

### CODE REVIEW

**Review by vogievetsky — APPROVED** [review]

File: processing/src/main/java/org/apache/druid/indexer/Checks.java [hunk 1]
```diff
@@ -36,12 +37,18 @@ public static <T> Property<T> checkOneNotNullOrEmpty(List<Property<T>> propertie
         if (nonNullProperty == null) {
           nonNullProperty = property;
         } else {
-          throw new IAE("At most one of %s must be present", properties);
+          throw new IAE(
+              "At most one of %s must be present",
```
Comment (kfaraz) [inline processing/src/main/java/org/apache/druid/indexer/Checks.java:41]: Nit: brackets for interpolation

```suggestion
              "At most one of [%s] must be present",
```

File: processing/src/main/java/org/apache/druid/indexer/Checks.java [hunk 2]
```diff
@@ -36,12 +37,18 @@ public static <T> Property<T> checkOneNotNullOrEmpty(List<Property<T>> propertie
...
+              properties.stream().map(Property::getName).collect(Collectors.toList())
```
Comment (kfaraz) [inline processing/src/main/java/org/apache/druid/indexer/Checks.java:42]: Maybe String.join() the items using ' or ' .

**Review by kfaraz — APPROVED** [review]
LGTM, minor message suggestions.

File: processing/src/main/java/org/apache/druid/indexer/Checks.java (hunk 2 above)
Comment (gianm) [reply processing/src/main/java/org/apache/druid/indexer/Checks.java:42]: IMO, with the interpolation it's clear, it'll look like:

```
At most one of properties[[a, b]] must be present
```

File: processing/src/main/java/org/apache/druid/indexer/Checks.java (hunk 1 above)
Comment (gianm) [reply processing/src/main/java/org/apache/druid/indexer/Checks.java:41]: 👍  Changed it to: `At most one of properties[%s] must be present` to also make it more obvious we're talking about properties.

**Pull request merged**